src/transcript/TranscriptEvent.ts

```diff
@@ -95,6 +96,14 @@ export class TranscriptEventConverter {
                 transcriptItem.vocabularyFilterMatch = item.vocabularyFilterMatch;
               }
 
+              if (item.stable) {
```

Comment:
`transcriptItem.stable` will be `undefined` if `item.stable` is `false`.

You should do one of the following to assign the `false` value:

```
if (typeof item.stable === 'boolean') {
```

or

```
if (typeof item.stable !== 'undefined') {
```

Reply:
Right. I will make the necessary changes for this. Thanks!!